packages/host/app/components/operator-mode/interact-submode.gts

```diff
@@ -218,9 +218,17 @@ export default class InteractSubmode extends Component<Signature> {
           }),
         );
       },
-      saveCard(card: CardDef, dismissItem: boolean): void {
+      saveCard: async (card: CardDef, dismissItem: boolean): Promise<void> => {
         let item = here.findCardInStack(card, stackIndex);
-        here.save.perform(item, dismissItem);
+        // WARNING: never await an ember concurrency perform outside of a task.
+        // Inside of a task, the `await` is actually just syntactic sugar for a
+        // `yield`. But if you await `perform()` outside of a task, that will cast
+        // the the task to an uncancellable promise which defeats the point of using
+        // ember concurrency.
+        // https://ember-concurrency.com/docs/task-cancelation-help
+        let deferred = new Deferred<void>();
+        here.save.perform(item, dismissItem, deferred);
+        await deferred.promise;
```

Comment:
@lukemelia this is a bit more complex implementation in order to tap dance around awaiting the perform and have an async interface. do you think it's worth the tradeoff? (make sure to note how the `this.save` has changed too)